tests/defaults.yml

```diff
@@ -1,3 +1,15 @@
 # Overwrite parameters here
 
-# parameters: {...}
+# parameters:
+#   topolvm:
+#     deviceclasses:
+#       - name: ssd
+#         volume-group: vgssd
+#         default: true
+#         spare-gb: 10
+
+#     storageclasses:
+#       ssd-local:
```

Comment:
I think must be adapted as well https://github.com/topolvm/topolvm/blob/main/charts/topolvm/values.yaml#L430.